eventsource-client/src/error.rs

```diff
@@ -18,8 +18,6 @@ pub enum Error {
     UnexpectedEof,
     /// Encountered a line not conforming to the SSE protocol.
     InvalidLine(String),
-    /// Encountered an event type that is not a valid UTF-8 byte sequence.
-    InvalidEventType(std::str::Utf8Error),
     InvalidEvent,
```

Comment:
This error is no longer possible. It will be caught when parsing the line, which would cause an `InvalidLine` error instead.